### DISCUSSION TITLE
Script to bridge TUS -> AVAX

### DISCUSSION THREAD

**Discussion by EugeneTeu** [coccoinomane/crabada.py#76] [Ideas]
Considering subnet coming in for idle game, we might want to have scripts to automate the following: 

## Scenario 1
1. bridge TUS out of subnet EVM -> c-chain
2. Swap TUS -> Avax on c-chain 

## Scenario 2
1. Swap TUS directly on subnet EVM  

I'm thinking that we likely will need to do scenario 1 since swimmer subnet might not have a DEX + i believe native currency on swimmer will be TUS. This might be useful for people who wants to sell TUS to cover gas / ROI

**Comment by coccoinomane** [comment]
For reference, an update on this topic:

- @EugeneTeu kindly implemented the swap feature in https://github.com/coccoinomane/crabada.py/pull/80. 
- The feature will not be implemented in crabada.py, because it is outside of its scope.
